src/text.ts

```diff
@@ -24,19 +25,41 @@ const getTwilioClient = () => {
   return client
 }
 
-export const sendText = async ({ body, to, logger }) => {
+export const sendTwilioText = async ({ body, to, logger }) => {
   try {
     await getTwilioClient().messages.create({
       from: twilioPhoneNumber,
       to,
       body,
     })
   } catch (err) {
-    logger.fatal({ err }, "impossible to send text")
+    logger.fatal({ err }, "impossible to send text through Twilio")
     return
   }
 
-  logger.info({ to }, "sent text successfully")
+  logger.info({ to }, "sent text through Twilio successfully")
```

Comment:
@krtk6160 will this match our current query in grafana of plotting the number of successful text being send over time?

Reply:
No I don't think it will. Good catch. We might need to add a field that represents a successful text msg (common across every send text log, independent of the provider)

Reply:
Do I need to change this logging message back to "sent text successfully" before we accept this PR?

Reply:
I'd suggest changing the text back, and then also adding something like `provider: xyz` so that we can differentiate between them in logs if needed

Reply:
yup better to have those variables in the first argument of the log function, and keep the message generic otherwise. this is how Pino should typically be used